handlers/handlers.js

```diff
@@ -122,11 +122,20 @@ const deregisterHandler = async (client, msg) => {
   }
 };
 
+const chatHandler = (client, text) => {
+  fs.readFile(../chatbot/phraselist.txt, function(err, data){
+    var lines = data.split('\n');
```

Comment:
There are lots of issues with using the keyword `var`. Use `let` for variables you will reassign (in the manner of `let lines; lines = somethingNew;`. Use `const` for variables you do not plan to reassign.

Reply:
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/